### PR TITLE
Added old and antique Biesel flags

### PR DESCRIPTION
**Pull request by Generalcamo**
Based on the flag by Caelphon.

### CODE REVIEW

**Comment by Generalcamo** [comment]
!review

**Comment by niennab** [comment]
Your missing the directionals.
directionals

File: code/game/objects/structures/flags_banners.dm [hunk 1]
```diff
@@ -1388,6 +1388,84 @@
...
+
+/obj/structure/sign/flag/biesel/old/large/east/New()
+	..(loc, EAST)
+
+/obj/structure/sign/flag/biesel/old/large/west/New()
+	..(loc, WEST)
+
+/obj/item/flag/biesel/antique
+	name = "antique Solarian Colonial Mandate of Tau Ceti flag"
+	desc = "The flag used by Biesel and Valkyrie before the Interstellar War, during its initial colonization. This flag still has the old Solarian Alliance canton in the corner. This flag is old enough that it is considered an antique and not illegal to display, but would still be a bad move to publically display it in Tau Ceti space, primarily due to its rarity."
```
Comment (listerla) [inline code/game/objects/structures/flags_banners.dm:1432]:
```suggestion
	desc = "The flag used by Biesel and Valkyrie before the Interstellar War, during its initial colonization. This flag still has the old Solarian Alliance canton in the corner. This flag is old enough that it is considered an antique and not illegal to display, but would still be a bad move to publicly display it in Tau Ceti space, primarily due to its rarity."
```

File: code/game/objects/structures/flags_banners.dm [hunk 2]
```diff
@@ -1388,6 +1388,84 @@
...
+	flag_path = "biesel_antique"
+	flag_structure = /obj/structure/sign/flag/biesel/antique
+
+/obj/structure/sign/flag/biesel/antique
+	name = "antique Solarian Colonial Mandate of Tau Ceti flag"
+	desc = "The flag used by Biesel and Valkyrie before the Interstellar War, during its initial colonization. This flag still has the old Solarian Alliance canton in the corner. This flag is old enough that it is considered an antique and not illegal to display, but would still be a bad move to publically display it in Tau Ceti space, primarily due to its rarity."
```
Comment (listerla) [inline code/game/objects/structures/flags_banners.dm:1438]:
```suggestion
	desc = "The flag used by Biesel and Valkyrie before the Interstellar War, during its initial colonization. This flag still has the old Solarian Alliance canton in the corner. This flag is old enough that it is considered an antique and not illegal to display, but would still be a bad move to publicly display it in Tau Ceti space, primarily due to its rarity."
```

**Review by niennab — APPROVED** [review]

**Review by FluffyGhoster — APPROVED** [review]

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**